app.js

```diff
@@ -101,30 +101,47 @@ const getTemperatures = () => {
   return temperatures;
 }
 
-//function pickHex(color1, color2, weight) {
-//  var w1 = weight;
-//  var w2 = 1 - w1;
-//  var rgb = [Math.round(color1[0] * w1 + color2[0] * w2), Math.round(color1[1] * w1 + color2[1] * w2), Math.round(color1[2] * w1 + color2[2] * w2)];
-//  return rgb;
-//}
-
 const drawPlots = () => {
-  const temperatures = getTemperatures();
-  const canvas = document.getElementById("xyGraph");
-  const ctx = canvas.getContext("2d");
+
+  const values = formValues();
 
   const steps = 500;
 
+  // declare vars
+  var size = 500, x = new Array(size), y = new Array(size), z = new Array(size), i, j;
+
+  // get x and y values
+  for (var i = 0; i < size; i++) {
+  x[i] = -values.x + (2 * values.x / size) * i;
+  y[i] = -values.y + (2 * values.y / size) * i;
+  z[i] = new Array(size)
+  }
+
+  // get thermal diffusivity
+  const alpha = values.thermalConductivity / (8000 * 460);
```

Comment:
I'd pull these into top level constants. Magic numbers are tough

Reply:
Ya, that was just temporary